package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "replaceholder",
-  "version": "1.0.0",
-  "description": "Replace placeholder images with the correct image if it exists.",
+  "version": "0.0.2",
```

Comment:
You package.json was created at the moment I forked your branch, and version here isn't aligned with your version in bower.js (which I took as the default). In your bower file version number is 0.0.1, so I've changed it to 0.0.2. So which one would you prefer?